packages/next/server/render.tsx

```diff
@@ -388,6 +389,10 @@ function createFlightHook() {
   }
 }
 
+function escapeJSONForFlightScript(input: mixed): string {
```

Comment:
```suggestion
function escapeJSONForFlightScript(input: unknown): string {
```
Should we use `unknown` or `any` here to fix https://github.com/vercel/next.js/runs/6473760106?check_suite_focus=true#step:5:220